### PR TITLE
{bio}[iccifort/2019.5.281] MethylDackel v0.5.0, libBigWig v0.4.4, HTSlib v1.10.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/6449aeb35bada819b0eeaa8085065368 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3152.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/f14709e5940533ecb671595a4d254d3d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2412.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/648eec7919a2e5f5db48c721f6629c16 for a full test report.

File: easybuild/easyconfigs/m/MethylDackel/MethylDackel-0.5.0-iccifort-2019.5.281.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+homepage = 'https://github.com/%(github_account)s/%(name)s'
+description = """A (mostly) universal methylation extractor for BS-seq experiments."""
+
+toolchain = {'name': 'iccifort', 'version': '2019.5.281'}
+
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+checksums = ['eecec7df109fbc468ea6bec538fb49ed92b13bddcbce7909fb23f792aaad349f']
+
+# both libBigWig library is statically linked
```
Comment (smoors) [inline easybuild/easyconfigs/m/MethylDackel/MethylDackel-0.5.0-iccifort-2019.5.281.eb]:
```suggestion
# the libBigWig library is statically linked
```

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node376.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/cf519bf260f48c67f710652a8dad8e9e for a full test report.

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**